doc/source/community/contributing.rst

```diff
@@ -322,24 +320,24 @@ You can find a pull request (or PR) tutorial in the `GitHub's Help Docs <https:/
 Style Guide & Linting
 ---------------------
 
-GeoPandas follows the `PEP8 <http://www.python.org/dev/peps/pep-0008/>`_ standard
-and uses `Black <https://black.readthedocs.io/en/stable/>`_ and
-`ruff <https://beta.ruff.rs/docs/>`_ to ensure a consistent code
-format throughout the project.
+GeoPandas follows the `PEP8 <http://www.python.org/dev/peps/pep-0008/>`_
+standard and uses `ruff <https://beta.ruff.rs/docs/>`_ to ensure a consistent
+code format throughout the project.
 
 Continuous Integration (GitHub Actions) will run those tools and
 report any stylistic errors in your code. Therefore, it is helpful before
 submitting code to run the check yourself::
 
-   black geopandas
+   ruff geopandas
```

Comment:
```suggestion
   ruff format geopandas
   ruff check geopandas
```
`ruff <path>` is deprecated and if you're replacing black, you want `ruff format` anyway